Makefile

```diff
@@ -113,12 +118,26 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	$(MAKE) -C conmon clean
 	$(MAKE) -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+# the approach here, rather than this target depending on the build targets
+# directly, is such that each target should try to build regardless if it
+# fails. And return a non-zero exit if _any_ target fails.
+local-cross:
+	@$(MAKE) --keep-going $(CROSS_BUILD_TARGETS)
+
+bin/crio.cross.%: .gopathok
+	@echo "==> make $@"; \
+	TARGET="$*"; \
+	GOOS="$${TARGET%%.*}" \
+	GOARCH="$${TARGET##*.}" \
+	$(GO) build -i $(LDFLAGS) -tags "containers_image_openpgp btrfs_noversion" -o "$@" $(PROJECT)/cmd/crio
```

Comment:
The Linux "cross compile" is [still failing][1].  Looks like we may need additional tags to turn off device-mapper and OSTree stuff?

[1]: https://travis-ci.org/kubernetes-incubator/cri-o/jobs/395210934#L1108

Reply:
Yes those should definitely be disabled.

Reply:
interesting that it doesn't fail on this for me locally :-\

Reply:
on the ancient Trusty version used by Travis the definition of `selabel_lookup_raw` is different.

on [Trusty](http://manpages.ubuntu.com/manpages/trusty/man3/selabel_lookup.3.html):
```
int
selabel_lookup_raw(struct selabel_handle *hnd,
                   security_context_t *context,
                   const char *key,
                   int type);
```

a more [recent release](http://manpages.ubuntu.com/manpages/bionic/man3/selabel_lookup.3.html):

```
int
selabel_lookup_raw(struct selabel_handle *hnd,
                   char **context,
                   const char *key,
                   int type);
```

Reply:
> on the ancient Trusty version used by Travis the definition of `selabel_lookup_raw` is different.

We need to set tags to avoid SELinux entirely.  Old Trusty packages are the tip of that iceberg, because OS X and Windows won't have SELinux libraries at all.

Reply:
but windows and darwin can avoid them by isolation that code to _linux.go files. It's trickier to "do the right thing" on fedora vs ubuntu

Reply:
> It's trickier to "do the right thing" on fedora vs ubuntu

Maybe for native builds.  But this cross-compile recipe is supposed to have a successful Linux bui;d on a Windows host, etc., and for that we want the straightforward "no SELinux at all".

Reply:
> The Linux "cross compile" is [still failing][1].

I've filed #1653 with additional build tags to help with this.

[1]: https://travis-ci.org/kubernetes-incubator/cri-o/jobs/395210934#L1108